pkg/storage/tsdb/index/chunk.go

```diff
@@ -4,7 +4,15 @@ package index
 type ChunkMeta struct {
 	Checksum uint32
 
-	// Time range the data covers.
-	// When MaxTime == math.MaxInt64 the chunk is still open and being appended to.
+	// Nanosecond precision
 	MinTime, MaxTime int64
+
+	// Bytes use an uint64 as an uint32 can only hold [0,4GB)
+	// While this is well within current chunk guidelines (1.5MB being "standard"),
+	// I (owen-d) prefer to overallocate here
+	// Since TSDB accesses are seeked rather than scanned, this choice
+	// should have little effect as long as there is enough memory available
+	Bytes uint64
+
+	Entries uint32
```

Comment:
Q: I assume this is number of log entries that are part of the particular chunk correct?. I'm trying to understand why `uint32` here and how it correlates with max size of each chunk?